shared_model/builders/protobuf/builder_templates/query_response_template.hpp

```diff
@@ -89,16 +89,22 @@ namespace shared_model {
       auto accountAssetResponse(
           const interface::types::AssetIdType &asset_id,
           const interface::types::AccountIdType &account_id,
-          const std::string &amount) const {
+          const interface::Amount &amount) const {
```

Comment:
What is the reason for the changes?

Reply:
Less error prone, also it is better to pass concrete object, instead of one of its representation